js/styles/chart.css

```diff
@@ -92,6 +92,7 @@ table.boxplotValues tr td
 }
 .axislabel, .axislabel text {
 	font-family: arial;
+	font-size: 1.2rem;

```

Comment:
What is style is '1.2rem'? Is that a real unit or just a typo from 'em'?

Reply:
Nevermind! rem is a real thing!

Reply:
rem is 'root em', which is calculated based on the font size of the body. 1rem = 100% body font-size, while 1em is 100% font-size of the parent element. I used it here to ensure that all labels have the same size